app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,9 @@
 class SessionsController < ApplicationController
   def new
-
+    if !session[:user_id].nil?
```

Comment:
Could we call current_user on this line since it's a helper method in our app controller?